doc/page-annotations.h

```diff
@@ -85,4 +85,31 @@ struct MyStruct
 }
 \endcode
 
+\section Annotations_impact Annotations impact
+
+Annotations can have different impact on IPC vs local methods of communications or C++ vs QML code.
+There is simplified description of this impact in table below. For detailed reference see above.
+
+|Annotation           | IPC | local | comments                 | C++ | QML| comments |
+|---------------------|-----|-------|--------------------------|-----|----|----------|
+|\@ipc-sync           |  +  | N/A   | generates ipc sync proxy | +/- | +  | method can be implemented in c++ but direct C++ usage requires extra effort |
```

Comment:
I do not understand what "method can be implemented in c++ but direct C++ usage requires extra effort" means.
Please use the existing tables describing annotations and add whatever information you believe is missing.
Annotations apply to different kind of object (interfaces, structs) and should not be mixed together in one single table.